### DISCUSSION TITLE
Request for join team

### DISCUSSION THREAD

**Discussion by ThakurRahulSingh** [ROM-builders/temporary#14338] [Q&A]
Please note that i've changed my GitHub username StockAndroid to ThakurRahulSingh, to match my GitHub username and my Telegram username.

Some successful builds

1. https://cirrus-ci.com/build/5164504722964480
2. https://cirrus-ci.com/build/5325166388445184
3. https://cirrus-ci.com/build/5948980516356096
4. https://cirrus-ci.com/build/5683119490596864
5. https://cirrus-ci.com/build/5612891750531072
6. https://cirrus-ci.com/build/6472475113422848
7. https://cirrus-ci.com/build/4626609679368192

**Cherry-pick list**
1. https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common/commits/pex-13
2. https://github.com/ThakurRahulSingh/device_xiaomi_miatoll/commits/pex-13
3. https://github.com/ThakurRahulSingh/vendor_xiaomi_sm6250-common/commits/12

Here my trees

Device Tree https://github.com/ThakurRahulSingh/device_xiaomi_miatoll
Device Tree-Common https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common

Here's my GitHub A/C http://github.com/thakurRahulSingh
Here's my TG A/C https://t.me/RahulSinghBhadoriya

**Comment by ThakurRahulSingh** [comment]
@Apon77 bro check please, if anything wrong then let me know

**Reply by ThakurRahulSingh** [reply]
Pls Reply bro

**Reply by Apon77** [reply]
please follow #learn instruction in telegram
you device tree says you know very little about git command line
